src/modules/tas_studio/editor/mod.rs

```diff
@@ -3879,43 +3895,56 @@ fn unwrap_angles(xs: impl Iterator<Item = f32>) -> impl Iterator<Item = f32> {
     })
 }
 
-fn smoothed_yaws(
```

Comment:
Instead of doing the same operations twice on two values (error-prone), pass the angles to work with explicitly. For example, move `let yaws = frames.iter().map(|f| f.state.prev_frame_input.yaw);` outside the function and accept it as an argument. Then the function can be called separately for pitch and yaw as needed.